tenv.go

```diff
@@ -211,3 +228,33 @@ func checkSelectorExprTarget(typ *ast.SelectorExpr) bool {
 	targetName := x.Name + "." + typ.Sel.Name
 	return targetName == "testing.TB"
 }
+
+// goVersionLower117 returns true if version is lower than Go 1.17 or empty.
+// version must be in the format 'go1.17', '1.17'.
+// In case of an invalid input returns not-nil error.
+func goVersionLower117(version string) (bool, error) {
+	version = strings.TrimPrefix(version, "go")
+	if version == "" {
+		return false, nil
+	}
+
+	parts := strings.Split(version, ".")
+	if len(parts) != 2 {
+		return false, errors.New(`go version must has format "go<MAJOR>.<MINOR>" or "<MAJOR>.<MINOR>"`)
+	}
+
+	major, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return false, fmt.Errorf("go version major part must be a number: %w", err)
+	}
+	if major < 1 {
```

Comment:
If Go become Go2 in the future (probably not), this linter must doesn't work. So I think this doesn't need. What do you think ?

Reply:
Let's remove this check, thanks.